src/commands/logic-app/createProject.ts

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as fse from "fs-extra";
+import * as vscode from "vscode";
+import { ext } from "../../extensionVariables";
+import { openFolder, selectWorkspaceFolder } from "../../utils/workspaceUtils";
+
+export async function createProject(): Promise<void> {
```

Comment:
Should this do something more? When I tested it out, I thought the extension had crashed because it dumped me to an empty workspace (I was already in an empty one, so it didn't look like anything changed). Like should we create a dummy/sample file in the folder to get people started?

Reply:
Perhaps it's just that I'm going into trying it out with no documentation

Reply:
An option could be that we stay on the extension page but give a pop up at lower right with button to "go to workspace"?

Reply:
I will add a progress notification for when this completes.